CMakeLists.txt

```
@@ -25,9 +25,10 @@ option(DPCTL_GENERATE_COVERAGE_FOR_PYBIND11_EXTENSIONS
     "Build dpctl pybind11 offloading extensions with coverage instrumentation"
     OFF
 )
-option(DPCTL_TARGET_CUDA
-    "Build DPCTL to target CUDA devices"
-    OFF
+set(DPCTL_TARGET_CUDA
+    ""
+    CACHE STRING
+    "Build DPCTL to target CUDA devices. Set to a truthy value (e.g., ON, TRUE) to use default architecture (sm_50), or to a specific architecture like sm_80."
```

Or is it possible to pass several value to `DPCTL_TARGET_CUDA` variable?
```suggestion
    "Build DPCTL to target CUDA device. Set to a truthy value (e.g., ON, TRUE) to use default architecture (sm_50), or to a specific architecture like sm_80."
```